api/apps/api/src/filters/all-exceptions.exception.filter.ts

```diff
@@ -24,14 +25,20 @@ import { AppConfig } from '../utils/config.utils';
 export class AllExceptionsFilter implements ExceptionFilter {
   private readonly logger = new Logger();
 
-  catch(exception: Error, host: ArgumentsHost) {
+  private isHttpError(exception: Error | HttpError): exception is HttpError {
```

Comment:
tiny: I would even avoid the `| HttpError` bit here - in my view, it doesn't really matter. When the function is called we know we have an `Error` of some kind, and then we do some narrowing in the chained ternary operator contraption further down.